Manage Organizations for MSSPs - CSIEM (#5720)
SumoLogic/sumologic-documentation: docs/manage/manage-subscription/create-and-manage-orgs/manage-orgs-for-mssps-csiem-rules.md

---
id: manage-orgs-for-mssps-csiem-rules

> title: Manage Organizations for MSSPs - CSIEM
> sidebar_label: Manage Orgs for MSSPs - CSIEM
> description: Learn how to manage organizational CSIEM rules and rule tuning for Managed Security Service Providers (MSSPs).

Reviewer: ```suggestion
title: Manage Organizations for MSSPs - Cloud SIEM
sidebar_label: Manage Orgs for MSSPs - Cloud SIEM
description: Learn how to manage organizational Cloud SIEM rules and rule tuning for Managed Security Service Providers (MSSPs).
```

> description: Learn how to manage organizational CSIEM rules and tuning epression for Managed Security Service Providers (MSSPs).

Author: ```suggestion
description: Learn how to manage organizational CSIEM rules and rule tuning for Managed Security Service Providers (MSSPs).
```

---
import useBaseUrl from '@docusaurus/useBaseUrl';

> This article describes how to manage organizational Cloud SIEM rules and rule tuning expression for Managed Security Service Providers (MSSPs). MSSP administrators must ensure that the content of their child organizations is properly configured. MSSPs often consist of a parent organization with child organizations that use [Cloud SIEM](/docs/cse/).

Reviewer: ```suggestion
This article describes how to manage Cloud SIEM rules and rule tuning expressions in organizations for Managed Security Service Providers (MSSPs). MSSP administrators must ensure that the content of their child organizations is properly configured. MSSPs often consist of a parent organization with child organizations that use [Cloud SIEM](/docs/cse/).
```

## Considerations

### Roles

You must have the following [organization role capabilities](/docs/manage/users-roles/roles/role-capabilities/#organizations) to create and manage organizations as an MSSP administrator:

* Organizations
   * View Organizations
   * Create Organizations
   * Manage Organizations

### Multi-insights list page in Cloud SIEM

> If you are logged in to a parent organization with child organizations that also use Cloud SIEM, the insights list page in Cloud SIEM allows you to [view insights in child organizations](/docs/cse/get-started-with-cloud-siem/about-cse-insight-ui/#view-insights-in-child-organizations). 

Reviewer 2: ```suggestion
If you are logged in to a parent organization with child organizations that also use Cloud SIEM, the Cloud SIEM insight list allows you to [view insights in child organizations](/docs/cse/get-started-with-cloud-siem/about-cse-insight-ui/#view-insights-in-child-organizations). 
```

To be able to see insights in child organizations, add child organizations that use Cloud SIEM. Then when the parent organization user goes to their Cloud SIEM insights list page, all the child organizations' insights appear in the list.
-->

## Manage Cloud SIEM rules

To ensure that content is consistent across child organizations, use the **Manage Content** tab to push content in target organizations with content from a source organization.

You can push the following:
* Cloud SIEM [rules](/docs/cse/rules/)
* Cloud SIEM [rule tuning expressions](/docs/cse/rules/rule-tuning-expressions/)

1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Administration > Organizations**.<br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the main Sumo Logic menu, select **Organizations**. You can also click the **Go To...** menu at the top of the screen and select **Organizations**.
1. Select the **Manage Content** tab.
1. In the **Source Org** field, select the organization that will provide the source data to be pushed in other organizations.
1. In the **Content** bar, select the content to be pushed:

>    * **Cloud SIEM Rules**. For more information about Cloud SIEM rules, refer to the [Cloud SIEM Rules](/docs/cse/rules/).
>    * **Cloud SIEM Rule Tuning**. For more information about Cloud SIEM rule tuning expressions, refer to the [Rule Tuning Expressions](/docs/cse/rules/rule-tuning-expressions/).

Reviewer: ```suggestion
   * **Cloud SIEM Rules**. For more information about Cloud SIEM rules, refer to [Cloud SIEM Rules](/docs/cse/rules/).
   * **Cloud SIEM Rule Tuning**. For more information about Cloud SIEM rule tuning expressions, refer to [Rule Tuning Expressions](/docs/cse/rules/rule-tuning-expressions/).
```

1. Click **Push**. A **Pushing in progress** dialog is displayed.

### Tips

* If you select **All Child Organizations**, you can then select organizations to exclude, allowing you to push to all organizations except those you select:<br/><img src={useBaseUrl('img/manage/subscriptions/mssp-orgs-selected-organizations.png')} alt="Selected organizations" style={{border: '1px solid gray'}} width="300"/>
* When you push rule tuning expressions, select **Include Associated Cloud SIEM Rules** to push all the Cloud SIEM rules that the expressions are used on:<br/><img src={useBaseUrl('img/manage/subscriptions/mssp-orgs-sync-associated-rules.png')} alt="Include Associated Cloud SIEM Rules checkbox" style={{border: '1px solid gray'}} width="200"/>

### Limitations

- If an item with the same name exists in the target organization, it will be replaced.
- Once a push is initiated, it cannot be reversed. Administrators should carefully review their selections before updating.

> - If errors occur during the push, administrators must manually re-attempt failed push. To see failed push, use [View History](#view-history).

Reviewer 2: ```suggestion
- If errors occur during the push, administrators must manually re-attempt failed push. To see failed pushes, use [View History](#view-history).
```

## View push in the audit log​

You can view all content management push in the [Audit Event Index](/docs/manage/security/audit-indexes/audit-event-index/) by using the following query:

```sql
_index=sumologic_audit_events
| where eventname = "ContentSynced"
```

To see the results displayed the same as in [View History](#view-history), use the following query:

```sql
_index=sumologic_audit_events
| where eventname = "ContentSynced"
| orchestratorJob.id as job_sync_id
| operator.email as user_email
| parseDate(eventTime, "yyyy-MM-dd'T'HH:mm:ss.SSSXXX") as eventTimeInmilliseconds
| values(resourceIdentity.name) as content, values(resourceIdentity.type) as content_type, min(eventTimeInmilliseconds) as content_sync_job_time_ms, values(status) as all_status by job_sync_id, user_email
| if (contains(all_status, "Failed"), "Failed", "Success") as status
| sort by content_sync_job_time_ms
| formatDate(content_sync_job_time_ms, "yyyy-MM-dd") as content_sync_date
| formatDate(content_sync_job_time_ms, "HH:mm:ss") as content_sync_time
| fields content_sync_date, content_sync_time, job_sync_id, status, user_email, content_type, content
```

## FAQs

> * **What happens when a item with the same name already exists?**<br/>It will be replaced in the child organization.

Reviewer 2: ```suggestion
* **What happens when an item with the same name already exists?**<br/>It will be replaced in the child organization.
```
